wp-api.php

```diff
@@ -51,56 +43,6 @@ function paginate_links(array $response) : string {
   return ob_get_clean();
 }
 
-function disable_default_wp_search() {
-  if (!search_enabled()) {
-    // Search is currently disabled. Don't override any search functionality.
-    return;
-  }
-
-  add_action('parse_query', function(WP_Query $query) {
-    if ($query->is_search()) {
-      $query->is_search = false;
-      $query->set('sitka_search', true);
-    }
-  });
-
-  add_filter('template_include', function(string $template) {
-    // Use the theme override search template if there is one.
-    // Otherwise, fallback on the plugin template.
-    $searchTpl = get_template_directory() . '/sitka-insights/search.php';
-
-    // Hook up Timber fallback view, if supported.
-    $searchTpl = apply_filters('sitka/timber/search_template', $searchTpl);
-
-    global $wp_query;
-    if ($wp_query->get('sitka_search') && file_exists($searchTpl)) {
-      // Force a 200 OK response.
-      $wp_query->is_404 = false;
-      status_header(200);
-
-      // Override the current selected WP template.
-      return $searchTpl;
-    }
-
-    return $template;
-  });
-
-  add_filter('wp_title', function(string $title, $separator) {
-    global $wp_query;
-    if ($wp_query->get('sitka_search')) {
-      $searchTerms = $wp_query->get('search_terms') ?: [];
-
-      $title = sprintf(
-        'Search: %s %s %s',
-        implode(' ', $searchTerms),
-        trim($separator) ?: '|',
-        get_bloginfo('name')
-      );
-    }
-    return $title;
-  }, 10, 2);
-}
-
 function enqueue_scripts() {
   // enqueue dependencies
   wp_enqueue_script('jquery-ui-core');
```

Comment:
This is a small thing, and not even anything new as of this PR. But just throwing it out there: Since your `sitka-js` script declares them as dependencies, I don't think you technically have to enqueue jQuery UI Core and jQuery UI Autocomplete here. They should be automatically enqueued by your script.

But then again, it doesn't hurt anything to explicitly enqueue them either. So take it or leave it.

Reply:
oh, I didn't actually know that, but it makes sense. I do like the more explicit way though.